CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
+**Added**
+
+- SecureNN with int64 support has landed.
+- Cryptographically secure random numbers feature has been implemented but not integrated.
```

Comment:
maybe it's worth switching to the un-seeded secure random ops before releasing? would it require a lot? then we could avoid the "but not integrated" part

Reply:
Sure lets do it. And then we can remove the part from the README too.